tests/test_parsley.py

```
@@ -1,97 +1,399 @@
+        bit_str.push(b'\x12\x34\x56\x78', 32)  # 32 bits of data
+        bit_str.push(b'\x9A\xBC', 16)          # 16 bits of data
+        msg_sid, msg_data = parsley.parse_bitstring(bit_str)
+        
+        # First 29 bits of 0x123456789ABC = 0x02468ACF
+        # Remaining 19 bits = 0x9ABC
+        assert msg_sid == b'\x02\x46\x8a\xcf'
+        assert msg_data == b'\x00\x9a\xbc'
+        
+    def test_parse_bitstring_empty(self):
+        bit_str = BitString() #just an empty bitstring
+        with pytest.raises(IndexError) as e:
```

Ruff (0.14.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)